packages/flutter/lib/src/cassowary/param.dart

```diff
@@ -6,45 +6,66 @@ import 'equation_member.dart';
 import 'expression.dart';
 import 'term.dart';
 
+/// Represents a variable inside the layout solver.
 class Variable {
   static int _total = 0;
 
+  /// Create a new variable with the given constant value.
   Variable(this.value) : _tick = _total++;
 
   final int _tick;
 
+  /// The current value of the variable.
   double value;
 
+  /// An optional name given to the variable. This is useful in debugging
+  /// solver state.
   String name;
 
+  /// Variables represent state inside the solver. This state is usually of
```

Comment:
this paragraph would be a good paragraph to have in the dartdocs for the whole class, but doesn't really say what `owner` is. Maybe:
```
/// An entity from outside the [Solver] that associated itself with this variable.
```